.github/workflows/e2e-matrix.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
     with:
       # This version matches the updated create-cluster action, which is now setup-cluster 
```

Comment:
This comment should be updated to reflect the new commit, probably makes sense to link the commit in the PR too.